api/src/main/resources/workbench.yaml

```diff
@@ -801,6 +801,40 @@ paths:
           description: A collection of participants
           schema:
             $ref: "#/definitions/CohortReview"
+    patch:
+      tags:
+        - cohortReview
+      description: Modifies the ParticipantCohortStatus status
```

Comment:
If this is going to be a patch of the cohort review resource, I would change the description to "Modifies the cohort review" and change the body to be ModifyCohortReviewRequest and response to be ModifyCohortReviewResponse (which can each contain a single ParticipantCohortStatus for now.)

This would allow us to modify other things about the cohort review in future without making a breaking change.

If you want to keep things this way -- a single ParticipantCohortStatus in the request and response -- I'd instead change the path to be something like:

/api/v1/workspaces/{workspaceNamespace}/{workspaceId}/review/{cohortId}/{cdrVersionId}/status/{participantId}

and have a PUT for that.

Reply:
Currently the client never has a case that will allow for updating anything on the CohortReview object. The client will only update the status, which may change things in the cohort_review table, but the client need not know or care about this. So i changed it to be a PUT --> /api/v1/workspaces/{workspaceNamespace}/{workspaceId}/review/{cohortId}/{cdrVersionId}/status/{participantId}